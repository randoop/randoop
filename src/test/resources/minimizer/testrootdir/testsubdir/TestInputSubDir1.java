package test.minimizer.testrootdir.testsubdir;

import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInputSubDir1 {
  // Test simplification of a program within a non-default package
  @Test
  public void test1() throws Throwable {
    String s = "String";
    org.junit.Assert.assertTrue(s.equals("String"));

    s = s.replace('i', 'o');

    // Fails, should be 'Strong'
    org.junit.Assert.assertTrue(s.equals("String"));
  }
}
