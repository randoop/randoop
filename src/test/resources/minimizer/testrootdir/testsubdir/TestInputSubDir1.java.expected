package test.minimizer.testrootdir.testsubdir;

import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInputSubDir1Minimized {

    // Test simplification of a program within a non-default package.
    // This test input file is located in a subdirectory and non-default package.
    // Checks that the minimizer is able to find the input file and minimize it.
    @Test
    public void test1() throws Throwable {
        String s = "String";
        s = s.replace('i', 'o');
        // Fails, should be 'Strong'
        org.junit.Assert.assertTrue(s.equals("String"));
    }
}
